Simpler focus on reference id without adding button element

a2line opened the pull request:
Restore templm original behavior in its trailer

a2line committed:
Simpler focus on reference id without adding button element

Restore templm original behavior in its trailer

fablhx commented:
Trigger AppVeyor CI

fablhx commented on src/util.ml:
```diff
@@ -964,7 +964,8 @@ value wprint_geneweb_link conf href s =
 value reference conf base p s =
   if conf.cancel_links || is_hidden p then s
-  else "<a href=\"" ^ commd conf ^ acces conf base p ^ "\">" ^ s ^ "</a>"
+  else "<a href=\"" ^ commd conf ^ acces conf base p ^ "\" 
```
newline after `else`

fablhx commented on hd/etc/templm/trl.txt:
```diff
@@ -0,0 +1,25 @@
+%if;(wizard and (evar.m="MOD_IND_OK" or evar.m="MOD_FAM_OK"))
```
Careful, your file is in DOS mode ! Does not work for templates

fablhx replied:
You may want to check the code base, because I did not take care about this since the bootstrap PR :)

a2line replied:
You're right! Done, in a second commit for all other template files.

a2line committed:
Clean format of template files

* remove CRLF on miss_evt (+ trim trailing spaces), perso_header and robots with dos2unix
* remove BOM from 5 text files with sed, linux/cygwin cmd is:
  sed -i '1 s/^\xef\xbb\xbf//' *.txt
* remove useless 2 eacute/2 nbsp caracters in perso_module/arbres, arbre_7gen, data3col
  (file were seen as ISO-8859 instead of ASCII)
* add a few missing end lines (only theter.min.js left without it, as release don't have it)

fablhx commented:
merged
